bsdlib/doc/CHANGELOG.rst

```diff
@@ -5,6 +5,17 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+bsdlib 0.6.1
+************
+
+* Implement hostname verification.
+* Report correct number of avaiable sockets for the application.
+* Simplify documentation in bsd_limits.h, and remove unused macros.
```

Comment:
~Simplify documentation in bsd_limits.h~ Removed unused `BSD_MAX_IP_SOCKET_COUNT` and `BSD_MAX_AT_SOCKET_COUNT` macros.

Reply:
ok